src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderCursor.cs

```diff
@@ -514,7 +514,7 @@ private void ThreadProc()
                                     if (_abort)
                                         return;
 
-                                    text = rdr.ReadLine();
+                                    text = rdr.ReadEntry();
```

Comment:
Should check if moving to multi-line reading will interfere with multi-threaded reading.

Reply:
How to check that, could you expand a bit?